templates/common/on-prem/files/NetworkManager-resolv-prepender.yaml

```diff
@@ -52,14 +52,21 @@ contents:
             "{{ onPremPlatformIngressIP . }}")
         DOMAIN="{{.DNS.Spec.BaseDomain}}"
         if [[ -n "$NAMESERVER_IP" ]]; then
-            >&2 echo "NM resolv-prepender: Prepending 'nameserver $NAMESERVER_IP' to /etc/resolv.conf (other nameservers from /var/run/NetworkManager/resolv.conf)"
-            sed -e "/^search/d" \
-                -e "/Generated by/c# Generated by KNI resolv prepender NM dispatcher script\nsearch $DOMAIN\nnameserver $NAMESERVER_IP" \
-                /var/run/NetworkManager/resolv.conf > /etc/resolv.tmp
+            source /etc/os-release
+            if [[ "$NAME" == "Fedora" ]]; then
+                echo "[Resolve]" > /etc/systemd/resolved.conf.d/kni.conf
```

Comment:
https://www.freedesktop.org/software/systemd/man/resolved.conf.html

I *think* we should use something like `60-kni.conf` to make ordering easier.

Reply:
Good idea, updated